homeassistant/components/wemo/light.py

```diff
@@ -73,7 +72,7 @@ def update_lights():
                 new_lights.append(lights[light_id])
 
         if new_lights:
-            add_entities(new_lights)
+            run_callback_threadsafe(hass.loop, async_add_entities, new_lights).result()
```

Comment:
We can use `hass.add_job` since it doesn't matter if we wait for the result. Adding entities is scheduled anyway.